cmp/cmpopts/example_test.go

```diff
@@ -0,0 +1,125 @@
+package cmpopts_test
```

Comment:
Please add license header.

Reply:
Done

Reply:
I know people aren't fond of 2020, but that's the current year ;)